### PR TITLE
feat(topics): add topics to all repos

### PR DESCRIPTION
**Pull request by arthurhenrique**
usage:
` ./set-topic-repos  'ACME-ORG' 'TOPIC' 'squad-|my-private'`

sample:
image

### CODE REVIEW

File: add-topic-repos [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        # there is no more projects to check
+        return 0
+    fi
+
+    process_count=0
+    for repo in $repos; do
+        if [[ ! $repo =~ $exclude_pattern ]] ; then
+            set_topic_repo "$repo" "$topic" &
+        fi
+	process_count=$(($process_count+1))
```
Comment (alissonperez) [inline add-topic-repos]: It is mixed tabs/spaces here, could be good to review this in all file.

Comment (arthurhenrique) [reply add-topic-repos]: done thx

**Review by alissonperez — APPROVED** [review]

File: add-topic-repos [hunk 2]
```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+GITHUB_API=${GITHUB_API:-https://api.github.com}
+DEFAULT_ORGANIZATION=$1
+TOPIC=$2
+exclude_pattern="(protected|private)"
```
Comment (rasecoiac03) [inline add-topic-repos]: I didn't understand this fixed exclude pattern.

Comment (arthurhenrique) [reply add-topic-repos]: this is a regex sample, but I could remove

Comment (rasecoiac03) [reply add-topic-repos]: No no, it's good, leave it.. I think you could do receive it as a parameter. ❤️

Comment (arthurhenrique) [reply add-topic-repos]: Like this 3093e96?

**Review by rasecoiac03 — APPROVED** [review]

File: add-topic-repos (hunk 2 above)
Comment (rasecoiac03) [reply add-topic-repos]: yes, :)

**Pull request merged**